build/ci/check_demo_pr_checks.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash -xe
+set +o pipefail
+
+export passed_k8s_checks=true
+export passed_openshift_checks=true
+
+wait_for_checks_to_complete(){
+  community_operators_branch=$1
+  all_checks_passed=true
+  export repo_pr=`gh pr list --repo $github_csiblock_community_operators_repository | grep $community_operators_branch`
+  if [[ "$repo_pr" == *"$community_operators_branch"* ]]; then
+    sleep 5
+    while [ "$(gh pr checks $community_operators_branch --repo $github_csiblock_community_operators_repository | grep -i pending | wc -l)" -gt 0 ]; do
+      sleep 5
+    done
+    if [ "$(gh pr checks $community_operators_branch --repo $github_csiblock_community_operators_repository | grep -i fail | wc -l)" -gt 0 ]
```

Comment:
could you grep for "not pass" instead of "fail"? to avoid potential false positives

Reply:
I meant something like `grep -iv pass`

Reply:
done